workspace/test.js

```diff
@@ -1,13 +1,11 @@
-import { assert, config as chaiConfig } from "chai";
-chaiConfig.truncateThreshold = 0;
+// test.js
 
-import * as LC from "@codewars/lambda-calculus";
-import { solution } from "./files.js"; // /workspace/files.js
+import * as LC from "./lc-test.js";
+LC.config.truncateThreshold = 0;
+LC.configure({ purity: "Let", numEncoding: "Church" });
```

Comment:
I don't think this works. `lc-test.js` has `export * as LC from "@codewars/lambda-calculus";` so this needs to be `LC.LC.configure`.

See https://github.com/codewars/lambda-calculus/pull/64/files#diff-048f6c0b471ab2151a053dc74ea5e88ae31b8c871aee8b4f8582bda6a6e9cd78

Reply:
Having stuff in `LC.LC.` is not my intention

Reply:
I'm awfully sorry but I can't. I can suggest what files should look like in the end, but I don't know how to suggest a diff on a diff.

Reply:
You can just tell me what's wrong with that PR.

Reply:
Your PR changes things that are then changed again by my PR, but from a different starting point.

I fear that will break things.

Reply:
? We're not merging both. I'll close this to avoid confusion.

Reply:
Then we merge yours I guess, and then I think some more changes are necessary.

Reply:
Like what? It should contain the equivalent code now. I changed to export `chai.config` and doesn't contain the import bug in this one.

#64 should have everything to create a new image.